ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -43,7 +43,7 @@
  * which is either the only chunk in the case of a simple blob, or the metadata chunk in the case of composite blobs.
  */
 class GetBlobInfoOperation extends GetOperation {
-  private final OperationCompleteCallback operationCompleteCallback;
+  private final OperationCallback operationCallback;
```

Comment:
Could this be moved to the superclass, `GetOperation`? It is present in both `GetBlobInfoOperation` and `GetBlobOperation`.